### ISSUE TITLE
[Feature]: Adding Providers for kshow123 and dramacool

### ISSUE THREAD

**Issue by thebetauser** [movie-web/movie-web#671]
### What feature do you want to add?

Adding additional asian tv shows providers:
* kshow123
* dramacool

### Why do you want to have this feature?

I think fans of kdrama will love having multiple providers for watching shows

### Anything other details to share?

These are the same as KissAsian (same embedded player and hosting providers)

**Comment by castdrian** [comment]
yeah I'll collect some korean providers at some point, I'd recommend goplay though it's focus is korean media
